core/pipeline/auth0.py

```diff
@@ -1,56 +1,110 @@
+import logging
+
+import jwt
 from social_core.exceptions import AuthFailed
 from social_core.pipeline.partial import partial
 
+logger = logging.getLogger(__name__)
+
+
+# def debug_pipeline(strategy, backend, *args, **kwargs):
```

Comment:
Let's not leave commented out code behind. Either change it to `logger.debug` or drop it.